### PR TITLE
openvpn: rebuild against mbedtls-2.16.7

### PR DESCRIPTION
**Pull request by jkoderu-git**

### CODE REVIEW

File: srcpkgs/openvpn/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'openvpn'
 pkgname=openvpn
 version=2.4.9
-revision=3
+revision=5
```
Comment (ericonr) [inline srcpkgs/openvpn/template:4]: 4, not 5

That said, why is this required? DId `mbedtls` break ABI? This might require a warning on its template to warn about cases like this.

Comment (jkoderu-git) [reply srcpkgs/openvpn/template:4]: Revision number is my mistake.

It is required because I see packages rebuild on new version of ssl libraries and mbedtls was updated recently. I am new to void development, I do not know of ABI.

Comment (ericonr) [reply srcpkgs/openvpn/template]: We rebuild packages on LibreSSL updates when they break binary compatibility (ABI = application binary interface), which means that trying to load the dynamic library simply doesn't work (or worse, has subtle errors). mbedtls, on the other hand, didn't have such a breakage on this last update., so rebuilding openvpn because of it shouldn't be necessary.

Comment (jkoderu-git) [reply srcpkgs/openvpn/template]: Thank you for explaining.

So security or bugfix release are ignored when rebuilding?

Comment (ericonr) [reply srcpkgs/openvpn/template]: Usually yes.

Comment (mobinmob) [reply srcpkgs/openvpn/template]: AFAICT mbedTLS does not break ABI in the LTS branch: https://tls.mbed.org/tech-updates/blog/announcing-lts-branch-mbedtls-2.16
That is why I reverted the template to this branch: https://github.com/void-linux/void-packages/pull/20695
